float32 x0_2
float32 y0_2
float32 z0_2
---
float32 T1
float32 T2
